AppointmentReminders.Web/Scripts/bootstrap-datetimepicker.js

```diff
@@ -0,0 +1,2426 @@
+/*! version : 4.14.30
```

Comment:
This should not be in the repo. The minified version might be okay, but I'd lean towards using a CDN.